src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -77,6 +77,16 @@ const Body = BaseFormWithPolling.extend(Object.assign(
       BaseForm.prototype.triggerAfterError.apply(this, arguments);
       this.stopPolling();
 
+      // Wait 1 min before polling again & do not show error message when encounter rate limit error
+      if (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests') ||
+        error.responseJSON?.errorCode === 'E0000047') {
+        model.trigger('clearFormError');
```

Comment:
@pradeepdewda-okta is there a way to hide error message somewhere here? I can't seem to remove it

Reply:
Hiding error message can be discussed if the UI folks have different opinion